UltraStar Play/Assets/Scenes/Demos/CodeStyleDemo/ReadonlyVsNotReadonlyField.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+
+public class ReadonlyVsNotReadonlyField : MonoBehaviour
+{
+    // The value of x can be viewed in the Inspector in debug mode, although it is private.
+    private int x = 1;
+    // However, the value of y cannot be viewed in the Inspector in debug mode, because it is readonly.
+    private readonly int y = 2;
```

Comment:
I see your point, but should this really go into the master branch? This could also be discussed & solved in a gist or issue...